docs/guides/algorithmiq-tem.ipynb

```diff
@@ -15,134 +15,82 @@
-    "options | dict | Input options. See `Options` section for more details. | No | \\{\"optimization_level\": 3\\}\n",
-    "\n",
-    "**Options**\n",
-    "\n",
-    "A dictionary containing the options for the TEM. The dictionary should contain the following keys:\n",
-    "\n",
-    "Name | Type | Description  | Example\n",
-    "-- | -- | -- | --\n",
-    "backend_name | str | Name of the backend to make the query.| \"ibm_fez\"\n",
-    "simulate_with_noise_model | bool | A Boolean flag indicating whether to simulate the noisy circuit or not.| False\n",
-    "max_bond_dimension | int | The maximum bond dimension to be used for MPOs. | True\n",
-    "tem_compression_cutoff | float | The cutoff value to be used for MPOs. | 1e-16\n",
-    "num_max_shots | int | The maximum number of shots. | 10_000\n",
-    "num_randomizations | int | The number of randomizations to be used for gate twirling. | 32\n",
-    "mitigate_readout_error | bool | A Boolean flag indicating whether to perform QDT readout error mitigation or not. | 24\n",
-    "num_readout_calibration_shots | int | The number of shots to be used for QDT readout error mitigation. | 1000\n",
-    "logging_level | str | The logging level to be used for the TEM runner. | \"INFO\"\n",
-    "\n",
-    "**Returns**\n",
-    "\n",
-    "A PubResult containing the TEM mitigated result:\n",
-    "\n",
-    "\n",
-    "Name |Type | Description\n",
-    "-- | -- | --\n",
-    "data | DataBin | A DataBin containing the TEM mitigated observable and its standard error. The DataBin has the following fields: <ul><li>`observable`: The TEM mitigated observable value.</li><li>`observable_stderr`: The standard error of the TEM mitigated observable.</li></ul>\n",
-    "metadadata | dict | A dictionary containing additional results. The dictionary contains the following keys: <ul><li>`\"observable_non_mitigated\"`: The observable value without error mitigation.</li><li>`\"observable_non_mitigated_stderr\"`: The standard error of the result without error mitigation.</li><li>`\"observable_simulated\"`: If its computation is enabled in the TEM options, contains the result obtained by simulating the circuit with the learned noise.</li></ul>"
+    "The noise model is an accurate description of the noise on the device able to capture subtle features, including qubit cross-talk. However, noise on the devices can fluctuate and drift and the learned noise might not be accurate at the point in which the estimation is done. This might result in inaccurate results. "
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "73390a19",
+   "id": "2995da29",
    "metadata": {},
    "source": [
     "## Get started\n",
     "\n",
-    "Authenticate using your [IBM Quantum Platform API token](http://quantum.ibm.com/), and select the TEM as follows:"
+    "Authenticate using your [IBM Quantum&trade; Platform API token](http://quantum.ibm.com/), and select the TEM as follows:"
```

Comment:
```suggestion
    "Authenticate using your [IBM Quantum Platform API token](http://quantum.ibm.com/), and select the TEM as follows:"
```
Can remove this marking once the admonition at the top is restored (since there is a marking in the admonition)